### ISSUE TITLE
Cosmetic build options seem to not be used by that many

### ISSUE THREAD

**Issue by BlazeGuy2** [StrangeLoopGames/EcoSuggestions#160]
_From @Dreepa on April 3, 2018 12:48_

Observation:
Most houses on our server look ugly (as in, no windows, no roofs, etc), as people seem to favor to play efficiently, so the do not put SP into the aesthetic skills.
It seems that pure cosmetic options are less worth to people than the new game-play options and effects they get from other areas to spend the SP.

The approach of just using one single SP pool for both, game-mechanic progression and visual options might be leaving room for improvement. People could be spending time beautifying their houses with different blocks, while waiting for craft jobs to complete.

I have the feeling it is an activity that should always be available, without having the feeling of "wasting" your skillpoints.

_Copied from original issue: StrangeLoopGames/EcoIssues#6921_

**Comment by BlazeGuy2** [comment]
_From @styles784 on April 7, 2018 20:43_

On the one hand, the current system makes house-building a viable job

But on the other hand, I never played games like Minecraft to ask others to build for me - doing it myself is a large part of the fun. It's tedious to experiment with the additional block types when you have to get another player to place your blocks for you, especially if both players aren't in the same voice channel. And paying SP to unlock an additional specialization so you can do it yourself causes subsequent specializations to ramp up in cost, or costs the player 50% of the SP if they back out of the specialization to lower the cost of a more important one

**Comment by BlazeGuy2** [comment]
_From @MarjieAVolk on April 9, 2018 17:45_

> Building a staircase, roof, window, column... doesn't really matter what the construction material is.
> Raise the price a little and roll all of the tier buildings in together into a single, unified construction tree. Tier 1 construction, tier 2 construction, tier 3 construction. Separate skills, but all significantly more useful and more valuable than they currently are.
> 
> Noone on my server is taking them. Because we have actual needs and they're way too expensive.
-- @CuAnnan

**Comment by BlazeGuy2** [comment]
_From @CuAnnan on April 11, 2018 2:10_

On that note. The skill should allow the creation of placeable items. Having to get permissions on someone's house to put in the decorations disinclines people to do it as well.

**Comment by BlazeGuy2** [comment]
_From @zimmah on April 16, 2018 2:21_

I think having it a separate skill could allow for Carpenters and Masons to be useful, however, there needs to be a way to make a contract that specifically asks for a house to be build, re-build or upgraded in a specific way, so that the Mason/Carpenter knows what their client wants, and the client doesn't need to give them full permission on their deeds.        
   
So basically by creating a contract you could make a blueprint, and with the blueprint you can " free-build" a ghost version of your house the way you want it, and the mason/carpenter has to match it exactly to finish the contract.

This gives players the fun of designing their own house, the safety of contracts, and it creates a new type of job.

**Comment by D3nnis3n** [comment]
This has changed in the meanwhile, the building skills are no longer there.
